### ISSUE TITLE
a bioinformatics tool or method that allows you to extract gene IDs from an annotated transcriptome?

### ISSUE THREAD

**Issue by timase2021** [brentp/fishers_exact_test#46]
Is there a tool that allows you to obtain gene IDs from a list of annotated transcriptome ?

**Comment by dbolser** [comment]
Yes, there are several highly effective tools specifically designed to map and convert gene, transcript, and protein identifiers. The best option for you depends on whether you prefer a quick web-based tool, an automated programmatic script, or a comprehensive offline genome browser.

@brentp Please close.
